### DISCUSSION TITLE
general_contract_provisions are handled different in JSON template and CSV templates

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/hospital-price-transparency#89] [Q&A] [answered]
In the CSV templates the general_contract_provisions is just one string. In the JSON format it is an array of object each of which has a payer_name, plan_name and  provisions.

It would be better if the three templates represented the same information structure. 

The standard approach being used for payer-plan specific values could be used here too for the CSV format.
The field header in CSV could be general_contract_provisions|[payer_name]|[plan_name] and the value then would be the provision.

**Comment by RidenourK** [comment, marked as answer]
@nmayle Thank you for your suggestion that we consider future Hospital Price Transparency – Data Dictionary GitHub repository updates to align the "general_contract_provisions" data element across the three CMS templates.
